src/prepend.js

```diff
@@ -0,0 +1,87 @@
+import * as parents from "./parents.js";
+import {properties} from "./children.js";
+
+/**
+ * Create a new MemberExpression node by combining an object and property
+ * @param {object | string} object
+ * @param {object | string} property
+ * @param {*} [o]
+ * @param {boolean} [o.computed] Whether to use computed syntax (e.g. `foo[bar]`) or dot syntax (e.g. `foo.bar`)
+ */
+export default function prepend (object, property, o = {}) {
+	const prependedNode = _prepend(object, property, o);
+	// check for parent
+	const parentPath = parents.path(property);
+	if (parentPath) {
+		const {property, index, node: parent} = parentPath;
+		// replace parent[property] or parent[property][index] with prependedNode
+		if (index !== undefined) {
+			parent[property][index] = prependedNode;
+		}
+		else {
+			parent[property] = prependedNode;
+		} 
+	}
```

Comment:
This logic is to find the original node's parent and update it's pointer to point to the new prepended node. Which module would this logic best fit in? A while back I started working on a `parents.replace` function, do you still think that's an appropriate name for this functionality @LeaVerou ?

Reply:
I'm going to finish the `children.replace` function because the logic is complex enough that it should be it's own function

Reply:
I opened #53 which will allow us to get rid of all this code and instead call `children.replace(node, prependedNode)`